Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -1183,7 +1183,22 @@ private void OnDisable()
             CoreServices.InputSystem?.UnregisterHandler<IMixedRealityTouchHandler>(this);
             CoreServices.InputSystem?.UnregisterHandler<IMixedRealityPointerHandler>(this);
 
+            // Currently in editor duplicating prefab GameObject containing both TMP and non-TMP children inside the Scrolling Object Collection container causes material life cycle management issues
+            // https://github.com/microsoft/MixedRealityToolkit-Unity/issues/9481
+            // Thus we do not automatically destroy material controlled by Material Instance if the OnDisable comes from pasting in editor
+#if UNITY_EDITOR
+            if (!Application.isPlaying)
+            {
+                bool? isCalledFromPastingGameObject = new System.Diagnostics.StackFrame(1)?.GetMethod()?.Name?.Contains("Paste");
+                RestoreContentVisibility(!isCalledFromPastingGameObject.GetValueOrDefault());
```

Comment:
If we don't destroy the material when we copy/paste, does that mean i now have two buttons with identical materials that both report themselves as "(Instance)"?

Reply:
No, the two button will not be sharing the same material instance.